.github/workflows/puppet.yaml

```diff
@@ -36,12 +39,12 @@ jobs:
       actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           submodules: false
 
       - name: Setup Ruby
-        uses: ruby/setup-ruby@55283cc23133118229fd3f97f9336ee23a179fcf # v1.146.0
+        uses: ruby/setup-ruby@v1.192.0
```

Comment:
Why do we no longer need to include this long hash?

Reply:
The original was a copy paste job, and it's better to use version tag instead of opaque commit id